openmetadata-ui/src/main/resources/ui/src/components/SettingsSso/SSOConfigurationForm/SsoConfigurationFormArrayFieldTemplate.tsx

```diff
@@ -151,9 +197,19 @@ const SsoConfigurationFormArrayFieldTemplate = (props: FieldProps) => {
           status={hasError ? 'error' : undefined}
           tagRender={SsoCustomTagRenderer}
           value={value}
-          onBlur={() => props.onBlur(id, value)}
-          onChange={(value) => {
-            props.onChange(value);
+          onBlur={() => {
+            const convertedValue = isScopeField ? value.join(' ') : value;
+            props.onBlur(id, convertedValue);
+          }}
+          onChange={(newValue) => {
+            // Handle scope field conversion from array (UI) to string (backend)
+            const convertedValue = isScopeField
```

Comment:
we should move this out for the render and same with onBlur!

Reply:
addressed